src/Shikensu/Utilities.hs

```diff
@@ -81,11 +82,12 @@ lsequence list =
 -}
 (!~>) :: (FromJSON a, ToJSON a) => Metadata -> Text -> a
 (!~>) obj key =
-    case (obj ~> key) of
-        Just x  -> x
-        Nothing -> error <|
-            "Could not find the key `" ++ (Text.unpack key)         ++ "` " ++
-            "on the metadata object `" ++ (aesonObjectToString obj) ++ "` using (!~>)"
+  fromMaybe errorMessage (obj ~> key)
```

Comment:
Thanks! I know this function exists, but I liked my solution better because I find it more readable. That is, I find it clearer when I'm going to get that `error` if I use a `case` statement. Does that make sense?

Also, I don't like to use `where` in my code, I prefer just to stick with `let`. Total personal preference, sorry 😳